clear/main.py

```diff
@@ -545,6 +546,28 @@ def route(prefix, vrf, namespace):
         command += ['-n', str(namespace)]
     clicommon.run_command(command)
 
+@click.option('--all', '-a', is_flag=True, help='Delete also compressed logs')
+@cli.command()
+def logging(all):
+    """Clear logging files"""
+    log_path_arr = ["/var/log"]
```

Comment:
Can we put a check here to ensure `/var/log` directory exists before setting `log_path_arr`?

Reply:
done